src/java.base/share/classes/sun/security/rsa/RSACore.java

```diff
@@ -57,17 +56,8 @@
  */
 public final class RSACore {
 
-    /*
-     * Check whether native crypto is enabled with property.
-     *
-     * By default, the native crypto is enabled and uses the native
-     * crypto library implementation.
-     *
-     * The property 'jdk.nativeRSA' is used to enable Native RSA alone,
-     * and 'jdk.nativeCrypto' is used to enable all native cryptos (Digest,
-     * CBC, GCM, and RSA, and EC).
-     */
-    private static boolean useNativeRsa = true;
+    /* The property 'jdk.nativeRSA' is used to disable native RSA. */
+    private static boolean useNativeRsa = NativeCrypto.isAlgorithmEnabled("jdk.nativeRSA", "RSACore");
```

Comment:
This should be `final` like other similar fields.

Reply:
`useNativeRsa` could be set to `false` later, if the native implementation fails.